client/signup/jetpack-connect/install-step.jsx

```diff
@@ -7,20 +7,42 @@ import React from 'react';
  * Internal dependencies
  */
 import Card from 'components/card';
+import { preventWidows } from 'lib/formatting';
 
 export default React.createClass( {
 	displayName: 'JetpackInstallStep',
 
+	renderText() {
+		const { action } = this.props;
+		let { text } = this.props;
+
+		if ( ! action ) {
+			return (
+				<span>
+					{ preventWidows( text ) }
+				</span>
+			);
+		}
+
+		return (
+			<span>
+				{ text }
+				<span> </span>
```

Comment:
The outer span is because we need to wrap the text in a component since we're returning the text from a method.

> The link is a sentence in this case, and it should be allowed to word-wrap, but also not be left with a widow. 

That does make sense. The issue is that when we pass in the text to `preventWidows` we're not passing in purely text. We're passing in components (the action is a link). Which kind of gets us in to the first comment which talks about we don't run the `preventWidows` when there is an `action` present.

We could look at modifying `preventWidows` to allow component input as well, but that seems a bit like a hornet's nest.

The solution in this PR fixes the widows for all but the first install step.

If we're willing to redesign/refactor such that the action link isn't appended to the text, we could go ahead and run `preventWidows` on both.